.doc_gen/metadata/ec2_metadata.yaml

```diff
@@ -94,6 +112,15 @@ ec2_CreateSecurityGroup:
             - description:
               snippet_tags:
                 - EC2.dotnetv3.CreateSecurityGroupExample
+    Kotlin:

```

Comment:
I don't see all of the snippets in the list like I would expect, such as DescribeKeyPairs?

Reply:
Some are in single action logic that shows same thing and pulled into the code cat in the same manner.